README.md

```diff
@@ -29,18 +29,16 @@ project.
 
 The default password of the livebook is `nerves` (as the time of writing, if it does not work, please check the nerves_livebook project). 
 
-## Interact with elixir-nx
-### OpenCV.Mat to Nx.tensor
-`OpenCV.Nx` module detects whether you have `:nx` available or not, if yes, then `OpenCV.Nx.to_nx/1` will try to convert
-`OpenCV.Mat` to `Nx.tensor`; otherwise, it returns `{:error, ":nx is missing"}`.
```

Comment:
I changed this in another PR but forgot to update the README. Sorry!

Reply:
No worries! 😀